src/SFML/Network/IpAddress.cpp

```diff
@@ -62,8 +62,8 @@ std::optional<IpAddress> IpAddress::resolve(std::string_view address)
         return Any;
 
     // Try to convert the address as a byte representation ("xxx.xxx.xxx.xxx")
-    if (const std::uint32_t ip = inet_addr(address.data()); ip != INADDR_NONE)
-        return IpAddress(ntohl(ip));
+    if (auto ip = inet_addr(address.data()); ip != INADDR_NONE)
```

Comment:
```suggestion
    if (const auto ip = inet_addr(address.data()); ip != INADDR_NONE)
```

Reply:
Now that `swapEndian` no longer takes a mutable reference, we can put `const` back where it was removed.